### ISSUE TITLE
recover the images

### ISSUE THREAD

**Issue by egonw** [bioclipse/bioclipse.scripting#1]
The Overleaf version I am copying from does not seem to have the images...

**Comment by jonalv** [comment]
What is the status of this? Which images are missing?

**Comment by egonw** [comment]
Three ones from the introduction

**Comment by jonalv** [comment]
Hm I visit this page too rarely. I am lost, the introduction of what exactly?

**Comment by egonw** [comment]
Previous version of the book...

**Comment by jonalv** [comment]
of which book? Which three images?

**Comment by egonw** [comment]
You're pulling my leg now...

**Comment by jonalv** [comment]
No is it from the Pharmbio book or what? How will I know which images your are talking about?

**Comment by egonw** [comment]
![image](https://user-images.githubusercontent.com/26721/45370584-d6778f80-b5e8-11e8-99f9-f9edf32331e1.png)

**Comment by egonw** [comment]
Yes, it was in TeX... I emailed you about this some time ago. The task was just about moving the images from their previous location into this repo. If you have them, I'd be delighted if you could send them, and then I'll take care of the rest...

I wanted this move to Markdown, because it's easier to work with, and doesn't look as fancy, but is pretty nice still: https://bioclipse.github.io/bioclipse.scripting/

(sorry for being a PITA)

**Comment by jonalv** [comment]
No I think maybe I am, and I do want to help you but I still don't know
which images. But you don't have some version with the images then I guess?

// Jonathan

**Comment by egonw** [comment]
No, indeed I don't... the Overleaf (tex) version I have does not have them... (link on your Skype...)
